src/apiv2.ts

```diff
@@ -312,18 +304,7 @@ export class Client {
     return reqOptions;
   }
 
-  public async getHeaders(): Promise<Record<string, string>> {
-    const opts: InternalClientRequestOptions<unknown> = { method: "GET" };
-    this.addRequestHeaders(opts);
-    await this.addAuthHeader(opts);
-    const map: Record<string, string> = {};
-    for (const h of opts.headers!) {
-      map[h[0]] = h[1];
-    }
-    return map;
-  }
-
-  public async getAccessToken(): Promise<string> {
+  private async getAccessToken(): Promise<string> {
```

Comment:
Maybe it's better to yoink this as an exported helper - that way you don't have to re-implement it in your Client, and you don't have to directly export the private variables. Thoughts? It doesn't need anything from Client, so it should be simple to do.

then it looks kinda neat, that you could do  `apiv2.getAccessToken()` on a whim.